# Type of the devices required. This is included as
# a required safety measure.
string type

# Name of the devices required (name of the package). If the name is left undefined
# then the first known entry under the specified type is executed
string name

# name of the executable/launchable/runnable
string executable

---

# name of the package
string name

# name of the executable/launchable/runnable
string executable

# return the name of the topic
string topic

# return code
#		0: Success
#		1: Failure
#	 -1: Undefined
int64 code

# return message
string message
